openedx/core/djangoapps/user_api/management/commands/retire_user.py

```diff
@@ -34,67 +34,72 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument(
-            '--username',
+            '--user_file',
```

Comment:
I am always a little wary of changing an interface, are we sure no one has coupled to the current `--username` argument/ implementation?

Reply:
@dnuzz , I have checked the code to see if this command is used somewhere else or not. But to be on save side we might want to keep it. This I created to get feedback from people. I have reached out to different people to know if this is being already used somewhere but still waiting on response.